extract_data/metadata.py

```diff
@@ -1221,19 +1294,23 @@ def insert_paper_with_references_doi(doi):
 
         elif len(existing_papers) > 1:
             # Multiple papers found, need to merge
-            print(f"Multiple papers with DOI {doi} found in the database. Merging required.")
+            print(
+                f"Multiple papers with DOI {doi} found in the database. Merging required."
+            )
             merge_papers(existing_papers)
             return
 
     print("Searching for a matching paper with no DOI.")
-    matching_paper= find_matching_paper_doi(doi)
+    matching_paper = find_matching_paper_doi(doi)
```

Comment:
I don't understand the combination of the print statement with the function call.